website/pages/home/index.jsx

```diff
@@ -18,20 +25,20 @@ export default function Homepage() {
               size: 55617,
               url:
                 'https://www.datocms-assets.com/2885/1539894412-vault-bg.jpg',
-              width: 3196
+              width: 3196,
```

Comment:
There are quite a few formatting changes littered through the diff, apologies for the noise!